editor/src/clj/editor/gui.clj

```diff
@@ -1987,8 +1996,7 @@
                                                                   [:node-msgs :scene-node-msgs]
                                                                   [:node-overrides :template-overrides]]]
                                                    (g/connect or-scene from self to))
-                                                 (for [[from to] [[:template-prefix :id-prefix]
-                                                                  [:current-layout :current-layout]
+                                                 (for [[from to] [[:template-trivial-gui-scene-info :aux-trivial-gui-scene-info]
```

Comment:
`:id-prefix` and `:current-layout` of the imported scene is dictated by the referencing `TemplateNode`.